cypress/integration/unit_tests/color_function_util_test.js

```diff
@@ -99,54 +99,59 @@ describe('Unit tests for color function utility', () => {
     td.trigger('click');
     expect(colorFunctionEditor.is(':visible')).to.be.true;
     expect(writeToFirebaseStub).to.not.be.called;
-    expect(getColorFunction()['color']).to.equal('yellow');
+    expect(getColorFunction().color).to.equal('yellow');
 
     // update color
     $('#single-color-picker').val('red').trigger('change');
     expectOneFirebaseWrite();
-    expect(getColorFunction()['color']).to.equal('red');
+    expect(getColorFunction().color).to.equal('red');
     expect(td.children().length).to.equal(1);
     expect(td.children().first().css('background-color')).to.equal('red');
 
     // switch to continuous
     $('#CONTINUOUS-radio').trigger('change');
     expectOneFirebaseWrite();
     const continuousPropertyPicker = $('#continuous-property-picker');
-    expect(getColorFunction()['current-style']).to.equal(0);
-    expect(getColorFunction()['color']).to.equal('red');
+    let currentStyle;
+    let color;
+    ({currentStyle, color} = getColorFunction());
```

Comment:
nit: why not just say `let` here and avoid the separate declaration and assignment?